R/current_compile_log.R

```diff
@@ -0,0 +1,12 @@
+current_compile_log <- function(log_file_path = getwd(), base_name) {
+  no_log_file <- sum(file.exists(log_file_path, hidden.files = TRUE)) == 0
```

Comment:
Would `!file.exists(log_file_path, hidden.files = TRUE)` suffice here?

Reply:
yes ... whoops - following an old team's conventions here (not using `!` and `TRUE` on the same line)